pkg/commands/workload.go

```diff
@@ -476,9 +511,13 @@ func (opts *WorkloadOptions) Update(ctx context.Context, c *cli.Config, currentW
 	}
 	c.Printf("Update workload:\n")
 	c.Printf("%s\n", difference)
-	if !workload.IsSourceFound() {
-		c.Printf("NOTICE: no source code or image has been specified for this workload.\n\n")
+
+	if noticeMsgs := workload.GetNotices(ctx); len(noticeMsgs) != 0 {
```

Comment:
can this `if-for` logic be isolated in a function to be used by `Create` and `Update` functions?

Reply:
this for loop prints msgs so i would like to keep this in in main func for readability purpose. if the logic is complex and needs testing then i would be more inclined to create a separate func